src/rules/converters/space-before-function-paren.ts

```diff
@@ -1,11 +1,15 @@
 import { RuleConverter } from "../converter";
 
+function isExistedESLintOption(rule: string) {
+    return ["anonymous", "named", "asyncArrow"].includes(rule);
```

Comment:
For these three, we'll want to include something in `notices` explaining that they will no longer be checked.

https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/docs/Architecture.md#rule-converters

Reply:
Added notices.